src/main/java/net/dv8tion/jda/api/entities/User.java

```diff
@@ -225,4 +228,168 @@ default String getEffectiveAvatarUrl()
      */
     @Nonnull
     JDA getJDA();
+
+    /**
+     * Returns the {@link net.dv8tion.jda.api.entities.User.UserFlag Flags} of this user.
```

Comment:
```suggestion
     * Returns the {@link net.dv8tion.jda.api.entities.User.UserFlag UserFlags} of this user.
```

Probably change these to accommodate the recent commit